### PR TITLE
[QRF-176] Lena theme cookie notice button 'disappears' on hover

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

**Comment by MohammadxAli** [comment]
Hey @fgyimah! I'm adding @alpower to review this as well because I feel like the color contrast between the background and text is not good on hover.
This is what it looks like now:
image

I guess changing the button text color to `white` on hover can solve the issue.

**Comment by fgyimah** [comment]
> Hey @fgyimah! I'm adding @alpower to review this as well because I feel like the color contrast between the background and text is not good on hover. This is what it looks like now: image
> 
> I guess changing the button text color to `white` on hover can solve the issue.

@MohammadxAli good catch, will update to use white text on hover

File: modules/Layout/CookieConsentBar/CookieConsentBar.module.scss [hunk 1]
```diff
@@ -94,3 +94,9 @@
         width: 100%;
     }
 }
+
+.acceptButton {
+    &:hover {
```
Comment (MohammadxAli) [inline modules/Layout/CookieConsentBar/CookieConsentBar.module.scss:99]: Maybe applying this for `:active` and `:focus` states too?

Comment (fgyimah) [reply modules/Layout/CookieConsentBar/CookieConsentBar.module.scss]: thanks, applied

**Review by MohammadxAli — APPROVED** [review]
Nice!

**Pull request merged**